data-tool/scripts/transfer_cprd_corps.sql

```diff
@@ -581,15 +889,12 @@ from (select e.event_id, -- SELECT BY EVENT
              ci.FOREIGN_NME
       from event e
          , CORP_INVOLVED ci
-         , corporation c
-        , corporation c2
+         , corporation_cte c
+        , corporation_cte c2
```

Comment:
Need to keep using `corporation` for TING(c2) information as there are some XPRO TINGs